docs/xcm-config.json

```diff
@@ -1,62 +1,78 @@
 {
-    "relaychain": {
-        "bin": "/usr/local/bin/polkadot",
-        "chain": "rococo-local",
-        "nodes": [
-            {
-                "name": "alice",
-                "wsPort": 9944,
-                "port": 30444
-            },
-            {
-                "name": "bob",
-                "wsPort": 9955,
-                "port": 30555
-            },
-            {
-                "name": "charlie",
-                "wsPort": 9966,
-                "port": 30666
-            },
-            {
-                "name": "dave",
-                "wsPort": 9977,
-                "port": 30777
-            }
-        ]
-    },
-    "parachains": [
-        {
-            "bin": "/usr/local/bin/interbtc-parachain",
-            "id": "21",
-            "wsPort": 8844,
-            "port": 30666,
-            "flags": [
-                "--rpc-port=9980",
-                "--",
-                "--execution=wasm"
-            ]
-        },
-        {
-            "bin": "/usr/local/bin/interbtc-parachain",
-            "id": "22",
-            "wsPort": 8855,
-            "port": 30777,
-            "flags": [
-                "--rpc-port=9990",
-                "--",
-                "--execution=wasm"
-            ]
-        }
-    ],
-    "simpleParachains": [],
-    "hrmpChannels": [
-        {
-            "sender": "21",
-            "recipient": "22",
-            "maxCapacity": 8,
-            "maxMessageSize": 512
-        }
-    ],
-    "types": {}
-}
\ No newline at end of file
+	"relaychain": {
+		"bin": "/usr/local/bin/polkadot",
+		"chain": "rococo-local",
+		"nodes": [
+			{
+				"name": "alice",
+				"wsPort": 9944,
+				"port": 30444
+			},
+			{
+				"name": "bob",
+				"wsPort": 9955,
+				"port": 30555
+			},
+			{
+				"name": "charlie",
+				"wsPort": 9966,
+				"port": 30666
+			},
+			{
+				"name": "dave",
+				"wsPort": 9977,
+				"port": 30777
+			}
+		],
+		"genesis": {
+			"runtime": {
+				"runtime_genesis_config": {
+					"configuration": {
+						"config": {
+							"validation_upgrade_frequency": 1,
+							"validation_upgrade_delay": 1
+						}
+					}
+				}
+			}
+		}
+	},
+	"parachains": [
+		{
+			"bin": "/usr/local/bin/interbtc-parachain",
+			"id": "21",
+			"balance": "1000000000000000000000",
+			"nodes": [
+				{
+					"wsPort": 9988,
+					"port": 31200,
+					"name": "alice",
+					"flags": ["--rpc-port=9980", "--", "--execution=wasm"]
```

Comment:
Is the RPC port not already set above?

Reply:
If you're referring to line `8` or `9`, those are relay chain nodes while this is a parachain node. I'm not sure what the port on line `48` is about

Reply:
Yeah I was referring to the one above, anyway I'll merge this now so I can test locally.